README.md

```diff
@@ -39,6 +41,24 @@ Use `basic_report.rb --help` to get started. There are several different reports
 
 You can find older examples of data-gathering scripts using `git log -- runners` (to see files that used to be in the "runners" directory) and post-processing scripts in the "formatters" directory.
 
+
+### speed.yjit.org site
+
+After collecting some data with `basic_benchmark.rb` you can generate the html
+site (hosted at speed.yjit.org) with a simple command:
+
+`site/exe serve`
+
+This will move files from `./data/` into a `build` directory,
+generate all the html files, and start a web server where you can view the site at `localhost:8000`.
+
+Some of the reports are built using `lib/yjit-metrics` and have templates in
+`lib/yjit-metrics/report_templates`.
+The rest of the files that build the site are found beneath `site`.
+There are `erb` files to generate additional pages
+and the script that does all the file rendering in `site/_framework/`.
```

Comment:
Not your doing, and not for this PR to address, but I wish the templates were all under `./templates`. Easy to find from the root of the repo.

Shell scripts could similarly all be under a `./scripts` directory or something. Nested directory structures with non-descriptive names make it hard to find things.

Reply:
Yes :)